x/did/keeper/private_test.go

```diff
@@ -30,27 +32,60 @@ func TestVerifyDIDOwnership_SigVerificationFailed(t *testing.T) {
 
 	sig, _ := types.Sign(doc, docWithSeq.Sequence+11234, privKey)
 
-	_, err := VerifyDIDOwnership(doc, docWithSeq.Sequence, docWithSeq.Document, docWithSeq.Document.VerificationMethods[0].Id, sig)
+	_, err := VerifyDIDOwnership(doc, docWithSeq.Sequence, docWithSeq.Document, sig)
 	require.ErrorIs(t, types.ErrSigVerificationFailed, err)
 }
 
-func newDIDDocumentWithSeq(did string) (types.DIDDocumentWithSeq, crypto.PrivKey) {
+func newDIDDocumentWithSeq(id string) (types.DIDDocumentWithSeq, crypto.PrivKey) {
 	privKey := secp256k1.GenPrivKey()
 	pubKey := secp256k1util.PubKeyBytes(secp256k1util.DerivePubKey(privKey))
-	verificationMethodID := types.NewVerificationMethodID(did, "key1")
-	es256VerificationMethod := types.NewVerificationMethod(verificationMethodID, types.ES256K_2019, did, pubKey)
-	blsVerificationMethod := types.NewVerificationMethod(verificationMethodID, types.BLS1281G2_2020, did, []byte("dummy BBS+ pub key"))
-	verificationMethods := []*types.VerificationMethod{
-		&es256VerificationMethod,
-		&blsVerificationMethod,
+	verificationMethodID := types.NewVerificationMethodID(id, "key1")
+	verificationMethod := []ariesdid.VerificationMethod{
+		{
+			ID:         verificationMethodID,
+			Type:       types.ES256K_2019,
+			Controller: id,
+			Value:      pubKey,
+		},
+		{
+			ID:         verificationMethodID,
+			Type:       types.BLS1281G2_2020,
+			Controller: id,
+			Value:      []byte("dummy BBS+ pub key"),
+		},
 	}
-	verificationRelationship := types.NewVerificationRelationship(verificationMethods[0].Id)
-	authentications := []types.VerificationRelationship{
-		verificationRelationship,
+
+	authentication := []ariesdid.Verification{
+		{VerificationMethod: *ariesdid.NewVerificationMethodFromBytes(verificationMethodID,
+			types.ES256K_2019,
+			id,
+			pubKey), Relationship: ariesdid.Authentication},
+		{VerificationMethod: ariesdid.VerificationMethod{
+			ID:         verificationMethodID,
+			Type:       types.ES256K_2019,
+			Controller: id,
+			Value:      pubKey,
+		}},
+	}
+
+	createdTime := time.Now()
+
+	doc := &ariesdid.Doc{
+		Context:            []string{ariesdid.ContextV1},
+		ID:                 id,
+		VerificationMethod: verificationMethod,
+		Authentication:     authentication,
+		Created:            &createdTime,
 	}
-	doc := types.NewDIDDocument(did, types.WithVerificationMethods(verificationMethods), types.WithAuthentications(authentications))
+	docBz, _ := doc.JSONBytes()
+
+	document := &types.DIDDocument{
+		Document:         docBz,
+		DocumentDataType: "aries-framework-go@v0.1.8",
```

Comment:
We can define this string as a const somewhere.
Also, it'd be better to use `github.com/hyperledger/aries-framework-go/pkg/doc/did.Doc@v0.1.8`. This is something like:
```go
fmt.Sprintf(
  "%s.%s@%s",
  reflect.TypeOf(doc).PkgPath(),
  reflect.TypeOf(doc).Name(),
  version,
)
```
I'm still not sure where to put the version string, but let's put it at last.

Reply:
@audtlr24 
Let's go with `github.com/hyperledger/aries-framework-go@v0.1.8/pkg/doc/did.Doc`. This is the official notation in Go.